### DISCUSSION TITLE
ESP32 One 3.5inch RPi LCD (C) no image

### DISCUSSION THREAD

**Discussion by typhoonss92** [HASwitchPlate/openHASP#312] [Hardware]
So I know the screen works as I tested on a RPi4. The Screen lights up white but no images/text. If I log into the device I am able to get all the menu's and can see the screenshot showing the pages with the json code. All seems good except not outputting to the screen. 
I uploaded the esp32-one_st7796_full_4MB_v0.6.2 first then tried the newest nightly build firmware update but still same thing. 
From everything I have read it should be plug and play. I have dealt with nextions(esphome), 4d systems and even the new nspanel(tasmota). I feel Like I am missing something. Any help would be appreciated.

**Comment by fvanroie** [comment]
This combination should indeed work with the prebuilt firmware. Do you have a link to the ESP and display you have? Or a picture of the back of the display? I'm not aware of any revisions or changes to the hardware, so it should be plug-and-play.

Please post the details of the info page and initial boot log if possible. Thanks.

**Comment by typhoonss92** [comment]
Sorry for the delay my internet has been out for the past 24 hours.
The ESP32 one is waveshare and the lcd is a waveshare 3.5 revision C. here are some pics also

![20220323_220100411_iOS](https://user-images.githubusercontent.com/14281675/159805349-a32f10c4-1445-42c6-84bd-461b40b38d63.jpg)
![20220323_220111998_iOS](https://user-images.githubusercontent.com/14281675/159805353-1f834da7-92e6-440a-a7d8-87957a351749.jpg)
![20220323_220130181_iOS](https://user-images.githubusercontent.com/14281675/159805354-f3682c4b-4d0c-461d-87e3-9e318586bf60.jpg)
![20220323_220054428_iOS](https://user-images.githubusercontent.com/14281675/159805357-31525d23-fe6a-42fb-96a3-3231278fd3ff.jpg)
.

**Comment by fvanroie** [comment]
Thank you for the pictures. They confirm that you have the correct hardware.

Meanwhile the issue has been found. The display chip of the 3.5" is an ILI9486 instead of the ST7796 of the 4" model. I'm not sure where the info came from as I was sure I've had read _somewhere_ that they were both ST7796... It turns out they are not the same and I have updated the docs now.

So, I added the ILI9486 for the ESP One board to the [development builds](https://github.com/HASwitchPlate/openHASP/actions). You can find the latest build [here](https://github.com/HASwitchPlate/openHASP/actions/runs/2034464659).
Scroll down near the bottom of that page for the `waveshare.zip` artifact which contains both firmware files for both screens.
For the 3.5" you will need to flash `esp32-one_ili9486_full_4MB_v0.6.3-dev_dec2531.bin`.

**Reply by typhoonss92** [reply]
Thanks for all your hard work making this project and getting back to me quickly and having a img ready. 
worked great. Now I need to get it connected to Home-assistant and on the wall.

**Comment by typhoonss92** [comment]
I have one more question. I soldiered the pads for the backlight control. from the documentation I should be setting gpio26 in the backlight control dropdown but that option is not available. is it something else for the different ILI9486 instead of the ST7796?

**Reply by fvanroie** [reply]
Whoops... that's another oversight. I've added it in the newest build fe7db71

**Comment by typhoonss92** [comment]
seems to be another odd thing. I have openhasp installed in HA. Have backlight set to turn screen off after 120sec
In HA is shows backlight off but screen is still on. when I turn backlight on the screen turns off. Seems something is backwards here. I have another plate I configured with wt32-sc01 which works correctly. Any thoughts?

**Comment by wolffman122** [comment]
So I am running into the same issue just a white screen but I can get into the web interface.   I have a lolin-d32-pro and the screen is this one [Screen](https://www.amazon.com/gp/product/B08MQLSCRP/ref=as_li_qf_asin_il_tl?ie=UTF8&tag=thingiverse09-20&creative=9325&linkCode=as2&creativeASIN=B08MQLSCRP&linkId=fe9efcc549172bbd31c6bab6b8999fc6l).  I followed the guide here [video](https://www.amazon.com/gp/product/B08MQLSCRP/ref=as_li_qf_asin_il_tl?ie=UTF8&tag=thingiverse09-20&creative=9325&linkCode=as2&creativeASIN=B08MQLSCRP&linkId=fe9efcc549172bbd31c6bab6b8999fc6) for wiring, what do I have to change in the build to make this work?

**Reply by fvanroie** [reply]
Please post in the [Lolin D32 Pro topic](https://github.com/HASwitchPlate/openHASP/discussions/235) which binary you have flashed to the board. Thanks.
